### DISCUSSION TITLE
Policy file documentation?

### DISCUSSION THREAD

**Discussion by geky** [veracruz-project/veracruz#68] [General]
Is there documentation over the contents of the policy file?

I found this, but seeing as it's in the test-collateral folder I have a feeling it's not the primary documentation:
https://github.com/veracruz-project/veracruz/blob/main/test-collateral/policy.template

Seeing as the policy file is probably the main way users will interact/configure the system, I think it would be good to have some sort of standard reference users could refer to without diving into the codebase. This would also be a good way to show off some of the features of Veracruz.

Maybe this can be generated from rustdoc somehow?

**Comment by dominic-mulligan-arm** [comment]
No, though I think there should be (probably the Wiki is the best place to put it) --- the best documentation at present is that template file and the code that actually interprets the policy, which is not the best.  Do you want to open an issue?

**Reply by geky** [reply]
Here we go https://github.com/veracruz-project/veracruz/issues/77

There's a bit of overlap for the wiki vs markdown files. One benefit of committing markdown files is that they stay in the commit history. This is especially useful for this sort of spec-like document, since it would contain the details of the codebase at that time.

**Reply by dominic-mulligan-arm** [reply]
OK.  Do you want to open an issue?  I think this is something that definitely needs documenting.

**Reply by dominic-mulligan-arm** [reply]
Oops, nevermind, I see that you've already done that!

**Reply by geky** [reply]
Ah yep, in #77. I couldn't find a way to "close" or someone indicate the discussion was moved elsewhere, but I guess GitHub Discussions are still under development.
